cmds/docs.js

```diff
@@ -11,6 +11,11 @@ module.exports = {
       alias: 'p',
       describe: 'Publish to GitHub Pages',
       default: false
+    },
+    markdown: {
```

Comment:
This flag should also be present on the `release` command, as it generates docs as well.